acceptance/phase_test.go

```diff
@@ -363,3 +367,27 @@ func withoutDaemonFixtures(phaseTest *PhaseTest) {
 func withoutRegistry(phaseTest *PhaseTest) {
 	phaseTest.targetRegistry = nil
 }
+
+func copyFakeSboms(t *testing.T) {
+	goos := runtime.GOOS
+
+	// Check Target Daemon != runtime.GOOS
+	if goos == "darwin" {
+		goos = "linux"
+	}
+	buildLifecycleDir, err := filepath.Abs(filepath.Join("..", "out", fmt.Sprintf("%s-%s", goos, runtime.GOARCH), "lifecycle"))
+	if err != nil {
+		t.Log("Fail to locate lifecycle directory")
+	}
+
+	extensions := [3]string{".sbom.cdx.json", ".sbom.spdx.json", ".sbom.syft.json"}
+	components := [2]string{"lifecycle", "launcher"}
```

Comment:
I can see this is used in several places and also seems to be constant. Is it a valid suggestion to externalize it?

Reply:
Maybe something like: 

```=golang
func SBOMExtensions() []string {
   return [3]string{".sbom.cdx.json", ".sbom.spdx.json", ".sbom.syft.json"}
}

func SBOMComponents() []string {
   return [2]string{"lifecycle", "launcher"}
}
```